### ISSUE TITLE
Generate manifest or monitors in Kuma via Code

### ISSUE THREAD

**Issue by sumeet-zuora** [louislam/uptime-kuma#4026]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Generate manifest or monitors in Kuma via Code, can understand using UI we can setup all the URL/HTTP/DNS etc monitoring but we are looking to setup them as code? is there a terraform provider or configmap or custom config which we can load in Kuma to create them automatically.

### 🐻 Uptime-Kuma Version

1.23

### 💻 Operating System and Arch

AL2

### 🌐 Browser

Chrome

**Comment by CommanderStorm** [comment]
First of all: Please use the search functionality before asking questions which have been asked before.
Answering the same question for the 9th time is really labour-intensive ^^

- config files are unlikely to be accepted at the current place in time due to the needed dependencys: https://github.com/louislam/uptime-kuma/issues/3578#issuecomment-1751696927
- terraform: https://github.com/louislam/uptime-kuma/issues/210
  TLDR: 
  Currently, we don't offer an api. See #118 for further details.
The current attempt at adding such a functionality is https://github.com/louislam/uptime-kuma/pull/3854 and might be merged into `v2.1` if all goes well.
  
  You can find external tooling which does wrap around our internal web socket endpoints, see https://uptime-kuma-api.readthedocs.io/en/latest/

**Comment by CommanderStorm** [comment]
closing as question should be resolved/this be considered a duplicate
